AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_BloomAdded.py

```diff
@@ -0,0 +1,188 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+class Tests:
```

Comment:
2 blank lines above classes at the top of file instead of 1

"Blank Lines
Surround top-level function and class definitions with two blank lines."
reference: https://www.python.org/dev/peps/pep-0008/#blank-lines